client/app/email-domain/email/redirection/email-domain-email-redirection.controller.js

```diff
@@ -20,7 +20,6 @@ angular.module("App").controller(
         $onInit () {
             this.loading = {
```

Comment:
```exportCSV``` and ```redirections``` are assigne multiple time but never used.

Reply:
saw with u. used in view to disable buttons.